unstructured/partition/utils/sorting.py

```diff
@@ -41,10 +42,13 @@ def sort_page_elements(
 
     if sort_mode == SORT_MODE_XY_CUT:
         coordinates_list = [el.metadata.coordinates for el in page_elements]
-        boxes = [coordinates_to_bbox(coords) for coords in coordinates_list]
-        res: List[int] = []
-        recursive_xy_cut(np.asarray(boxes).astype(int), np.arange(len(boxes)), res)
-        sorted_page_elements = [page_elements[i] for i in res]
+        if any(coords is None for coords in coordinates_list):
+            sorted_page_elements = page_elements
```

Comment:
probably worth a warning, because i don't think should get called in the first place under normal assumptions